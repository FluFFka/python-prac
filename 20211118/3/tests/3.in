a = Alpha(q=2,w=4,g=2)
print(a)
a.q = '123'
print(a)
b = AlphaQ(q=2,w=4,g=2)
print(b)
b.r = 123
b.w = 'tt'
b.x = int
b.p = AttributeError
print(b)
