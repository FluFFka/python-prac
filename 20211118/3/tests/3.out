g: 2, q: 2, w: 4
g: 2, q: 123, w: 4
g: 2, q: 2, w: 4
g: 2, p: <class 'AttributeError'>, q: 2, r: 123, w: tt, x: <class 'int'>
